userPayeeEndorsement.go

```diff
@@ -236,6 +236,10 @@ func (upe *UserPayeeEndorsement) Validate() error {
 		msg := fmt.Sprint(msgInvalid)
 		return &FieldError{FieldName: "UserRecordFormatType", Value: upe.UserRecordFormatType, Msg: msg}
 	}
+	if err := upe.isNumeric(upe.FormatTypeVersionLevel); err != nil {
```

Comment:
There was a test that intended to ensure this validation was being performed, but the test was written such that no assertions were run if no error was returned.